horreum-web/src/domain/user/ApiKeys.tsx

```diff
@@ -110,7 +110,7 @@ export default function ApiKeys() {
 
     const performRenameApiKey = () => {
         if (renameKeyId) {
-            userApi.renameApiKey(renameKeyId, renameKeyName)
+            userApi.renameApiKey(renameKeyId, renameKeyName!)
```

Comment:
Not sure if this is the "correct" way of doing it, my ts skills are non existent :)

Reply:
I would rather add the check in the if stmt rather then adding `!` which could fail bad if the value is actually undefined

Reply:
```suggestion
            userApi.renameApiKey(renameKeyId, renameKeyName)
```